### PR TITLE
feat: eleventy framework preset

### PR DESCRIPTION
**Pull request by PabloDiehl**
Eleventy (or [11ty](https://www.11ty.dev)) is a SSG (static site generator).

### CODE REVIEW

**Comment by jcbsfilho** [comment]
@PabloDiehl Firstly, the preset was very good.
All that was missing in the presets test unit was adding the expect for the preset.

Expected:

```js

// https://github.com/PabloDiehl/vulcan/blob/stage/lib/utils/presets/presets.utils.test.js#L10

const expectedOutput = [
   ...
      'eleventy',
   ...
    ];
    
// https://github.com/PabloDiehl/vulcan/blob/stage/lib/utils/presets/presets.utils.test.js#L77

const expectedResult = [
    ...
    'Eleventy (Deliver)',
    ...
  ];

```

**Comment by PabloDiehl** [comment]
> @PabloDiehl Firstly, the preset was very good. All that was missing in the presets test unit was adding the expect for the preset.
> 
> Expected:
> 
> ```js
> // https://github.com/PabloDiehl/vulcan/blob/stage/lib/utils/presets/presets.utils.test.js#L10
> 
> const expectedOutput = [
>    ...
>       'eleventy',
>    ...
>     ];
>     
> // https://github.com/PabloDiehl/vulcan/blob/stage/lib/utils/presets/presets.utils.test.js#L77
> 
> const expectedResult = [
>     ...
>     'Eleventy (Deliver)',
>     ...
>   ];
> ```

Added the missing tests.

**Review by jcbsfilho — DISMISSED** [review]
Nice!

**Review by jotanarciso — CHANGES_REQUESTED** [review]

File: lib/presets/custom/eleventy/deliver/prebuild.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { rm } from 'fs/promises';
+import { exec, copyDirectory, generateManifest } from '#utils';
+
+const manifest = {
```
[the remark this answers is not in the document]
Comment (PabloDiehl) [reply lib/presets/custom/eleventy/deliver/prebuild.js:4]: 🤔 
Good question, probably we'll need this in production environment. ⚒️

**Review by jotanarciso — APPROVED** [review]

**Review by jcbsfilho — APPROVED** [review]

**Pull request merged**